pennylane/ops/op_math/controlled.py

```diff
@@ -569,6 +570,14 @@ def simplify(self) -> "Controlled":
         )
 
 
+def _is_single_qubit_special_unitary(op):
+    if not op.has_matrix or len(op.wires) != 1:
+        return False
+    mat = op.matrix()
+    det = mat[0, 0] * mat[1, 1] - mat[0, 1] * mat[1, 0]
```

Comment:
any reason for computing this manually?

Reply:
It worked and was easy?

Reply:
makes sense. i got curious and it's like 6x faster than `np.linalg.det` too!